src/webapp/components/DeliveryTable.js

```diff
@@ -5,28 +5,40 @@ import TableCell from "@material-ui/core/TableCell";
 import TableContainer from "@material-ui/core/TableContainer";
 import TableHead from "@material-ui/core/TableHead";
 import TableRow from "@material-ui/core/TableRow";
+import Tooltip from "@material-ui/core/Tooltip";
 import Paper from "@material-ui/core/Paper";
 import { makeStyles } from "@material-ui/core/styles";
 import { useTranslation } from "react-i18next";
 import ClusterMapContext from "../context/ClusterMapContext";
 import DeliveryTableRow from "./DeliveryTableRow";
+import { getDaysSinceIsoTimestamp } from "../helpers/time";
 
 const useStyles = makeStyles(() => ({
   container: {
     maxHeight: "90vh",
   },
+  headerTooltip: {
+    fontSize: "0.8rem",
+  },
 }));
 
 const TableHeadRow = () => {
+  const classes = useStyles();
   const { t: str } = useTranslation();
   return (
     <TableRow>
       <TableCell />
-      <TableCell>
-        {str("webapp:deliveryNeeded.table.headers.daysOpen", {
-          defaultValue: "Days open",
-        })}
-      </TableCell>
+      <Tooltip
+        title={str("webapp:deliveryNeeded.table.headers.daysOpen.tooltip")}
```

Comment:
Reviewer(s): I wasn't sure if I should include a `defaultValue`. I opted _not_ to because it feels redundant, but let me know if you'd like me to.

Reply:
I think in general we always have a defaultValue just in case the translation files are missing the copy. It does seem pretty redundant, I agree. Originally the translation files exist so that other mutual aids forking this app have an easy way to customize copy. I am still trying to figure out if this is still the case.

Usually when I'm working on the app, I write the `str` statement with the defaultValue and when you run `npm run find-new-strings` to add the translation entry, it automatically pulls the defaultValue in the translation files. There's documentation here but it's not very findable: https://github.com/crownheightsaid/mutual-aid-app/blob/783ee668a1e8acd992179105f88804053205f485/src/lib/strings/DEVELOPING.md

Maybe to be safe we can add the defaultValue for now.